src/cr/cube/dimension.py

```diff
@@ -416,6 +416,16 @@ def hidden_idxs(self):
             idx for idx, element in enumerate(self.valid_elements) if element.is_hidden
         )
 
+    @lazyproperty
+    def is_cat_date(self):
+        """True if the dimension is a categorical date, False otherwise.
+
+        A dimension is considered a categorical date if it:
+        1. is a categorical dimension in the first place and
+        2. if it has a special entry "date" in the dimension dictionary.
+        """
+        return any(el.has_date for el in self.all_elements)
+
```

Comment:
per previous comment maybe this logic can be moved al the dimension type inspection level

Reply:
yeah, if we can easily create a new type, sure. But I'm afraid of that new type, that we'll have to sprinkle a bunch of 
```
if dimension_type in (DT.CAT, DT.CAT_DATE):
   . . .
```
where we wouldn't really care in the first place...

Reply:
ok, refactored...